superset/config.py

```diff
@@ -194,6 +194,13 @@
 ENABLE_CORS = False
 CORS_OPTIONS = {}
 
+# Chrome allows up to 6 open connections per domain at a time. When there are more
+# than 6 slices in dashboard, a lot of time fetch requests are queued up and wait for
+# next available socket. PR #5039 is trying to allow domain sharding for Superset,
```

Comment:
`is trying to allow domain sharding for Superset` => `adds domain sharding to Superset`

Reply:
fixed.